app/mailers/user_mailer.rb

```diff
@@ -14,11 +14,9 @@ def welcome(user)
           :subject => "Welcome to #{@appname}! thanks for signing up")
   end
 
-  # def create(email, product, charge)
-  #   @appname = "Bearlin Bikes"
-  #   mail( :to => email,
-  #         :subject => "Thank you for buying in #{@appname}! Here is your order") do |format|
-  #     format.html { render 'mail_order' }
-  #   end
-  # end
+  def create(user, charge)
+    @appname = "Bearlin Bikes"
+    mail( :to => user.email,
```

Comment:
Better to use new Ruby hash syntax like this:

`mail(to: user.email, subject: "Thank you for buying in #{@appname}! Here is your order")`